planner/core/rule_topn_push_down.go

```diff
@@ -142,7 +142,7 @@ func (p *LogicalJoin) pushDownTopNToChild(topN *LogicalTopN, idx int) LogicalPla
 	newTopN := LogicalTopN{
 		Count:   topN.Count + topN.Offset,
 		ByItems: make([]*ByItems, len(topN.ByItems)),
-	}.Init(topN.ctx)
+	}.Init(topN.ctx, topN.blockOffset)
```

Comment:
Should the pushed down topn uses the same offset with its original one.

Reply:
I think so, if one day we have hints about topn, the corresponding block offset should be the original one in order to take effects.